app/src/main/java/org/oppia/android/app/home/HomeActivity.kt

```diff
@@ -40,8 +40,7 @@ class HomeActivity :
   private var internalProfileId: Int = -1
 
   companion object {
-
-    var shouldRecreateActivity: Boolean = false
+    var shouldRecreateHomeActivity: Boolean = false
```

Comment:
Hi Ben thought this would be better since it is a publicly accessible variable. But the former also makes much sense in similar manner that can be understood. Reverting this.